src/main/resources/logback.xml

```diff
@@ -0,0 +1,11 @@
+<configuration>
+    <root level="DEBUG">
+        <AppenderRef ref="STDOUT"/>
+    </root>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} %-5level - %msg%n</pattern>
+        </encoder>
+    </appender>
+</configuration>
```

Comment:
Underrated vscode setting

![Screenshot 2023-09-29 at 2 54 28 PM](https://github.com/Eppo-exp/java-server-sdk-bandit-beta/assets/2822951/82b9dec7-3809-43b8-bff0-5aa7c2f6c0f2)

Reply:
Oh I have that set vscode, was using IntelliJ (an underrated IDE 😛)--I'll set it there too!